app/models/communities.rb

```diff
@@ -0,0 +1,59 @@
+class Communities
```

Comment:
Why is this a model? It doesn't seem to act like one... As far as I can see you never actually even create an object of this class... Rather the class methods that look like they should be constructors for the model are actually just returning plain hashes?